x-pack/plugins/aiops/public/components/log_rate_analysis/log_rate_analysis_page.tsx

```diff
@@ -196,11 +303,22 @@ export const LogRateAnalysisPage: FC = () => {
               setSearchParams={setSearchParams}
             />
           </EuiFlexItem>
-          <LogRateAnalysisContent
-            embeddingOrigin={AIOPS_TELEMETRY_ID.AIOPS_DEFAULT_SOURCE}
-            esSearchQuery={searchQuery}
-            onWindowParametersChange={onWindowParametersHandler}
-          />
+          <EuiFlexItem>
+            <LogRateAnalysisContent
```

Comment:
Should we clear this info if the brushes are reset or when a new analysis is run?

Reply:
Added this to https://github.com/elastic/kibana/issues/181111. I think it makes sense to clear the reset AI assistant section when Reset is pressed or when a new analysis is run.